Swift/Tests/QueryTest.swift

```diff
@@ -265,6 +265,32 @@ class QueryTest: CBLTestCase {
         XCTAssertEqual(firstNames.count, 5);
     }
     
+    // https://issues.couchbase.com/browse/CBL-2036
+    func testFTSQueryOnDBNameWithHyphen() throws {
```

Comment:
Based on the test name, does the issue happen only with FTS?

Reply:
Yea, this happens with FTSQuery only, 
https://github.com/couchbase/couchbase-lite-core/blob/master/LiteCore/Query/QueryParser.cc#L317